RELEASE/scripts/autoscend.ash

```diff
@@ -2384,20 +2384,140 @@ boolean powerLevelAdjustment()
 	return false;
 }
 
+boolean unlockThinknerdWarehouse()
```

Comment:
This boolean function doesn't follow any of the calling conventions of the rest of the codebase. I prefer to use the `LX_` prefix for functions that potentially adventure. You can add a `boolean thinknerdWarehouseUnlocked()` for inspecting the world. You could use this as `LX_unlockThinknerdWarehouse()` followed by `if(thinknerdWarehouseUnlocked())`

Reply:
The reason I did not put an LX in front of it is because it does not adventure anywhere.
My TODO is to make a second function that does start with LX that adventures somewhere to get a shirt

Or do you mean that potentially spending an adv crafting counts?

Reply:
change made

boolean boolean unlockThinknerdWarehouse() replaced with
boolean LX_unlockThinknerdWarehouse(boolean allow_adv, boolean allow_wish)

although at the moment it is a bit of a misnomer as it does not adventure not does it wish. although it could spend an adventure on crafting.